src/Model/Ssr/SearchAdapter.php

```diff
@@ -35,16 +38,36 @@ public function __construct(
         $this->priceFormatter      = $priceFormatter;
     }
 
-    public function search(string $query = '*', array $params = []): array
+    public function search(string $paramString): array
     {
         $client = $this->clientBuilder
             ->withServerUrl($this->communicationConfig->getAddress())
-            ->withCredentials($this->credentialsFactory->create());
+            ->withCredentials($this->credentialsFactory->create())
+            ->withVersion($this->communicationConfig->getVersion())
+            ->build();
+
+        $endpoint = $this->createEndpoint($paramString);
+        $response = $client->request('GET', $endpoint);
+
+        if (!$response) {
+            throw new ClientException('The response was empty');
+        }
+
+        return $this->priceFormatter->format($this->searchResult($response));
+    }
 
-        $searchAdapter           = (new AdapterFactory($client, $this->communicationConfig->getVersion()))->getSearchAdapter();
-        $searchResult            = $searchAdapter->search($this->communicationConfig->getChannel(), $query, $params);
-        $searchResult['records'] = $this->priceFormatter->format($searchResult);
+    private function searchResult(ResponseInterface $response): array
```

Comment:
spacing

Reply:
where exactly?
cs doesn't show me anything

Reply:
now looks fine, probably it was an github issue